Tools/CISupport/Shared/maybe-delete-directory.bash

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Deletes the directory given as the first argument if and only if it is greater than 10GB in size
+
+if [[ $# < 1 ]]
+then
+    echo "Must provide absolute path to directory to prune."
```

Comment:
Nit: it would be nicer to print this out as "Usage" help,  as opposed to having the computer give orders to humans :)